guacamole/src/main/frontend/src/app/import/controllers/importConnectionsController.js

```diff
@@ -322,52 +318,6 @@ angular.module('import').controller('importConnectionsController', ['$scope', '$
         return $q.all({ ...userRequests, ...groupRequests });
     }
 
-    // Given a PATCH API response, create an array of patches to delete every
-    // entity created in the original request that generated this response
-    const createDeletionPatches = creationResponse =>
```

Comment:
I removed all this rollback functionality because:
* It would be very hard / impossible to effectively implement in some cases, like in the case of a connection being removed and then re-added to reset the permissions.
* It would be unlikely to work anyway - if something like a create user or grant permission call fails, it's most likely because of the browser lost connection to the backend, or the backend experienced some sort of error. In that case, the subsequent calls to put things back the way they were wouldn't work anyway.